worker/charts/worker/templates/env-vars.yaml

```diff
@@ -73,17 +73,9 @@ stringData:
     REDIS_MASTER: {{ .Values.redis.master }}
     {{- end }}
 
-    {{- if and .Values.redis .Values.redis.tls .Values.redis.tls.enabled }}
     REDIS_TLS_ENABLED: {{ .Values.redis.tls.enabled | default "false" | quote }}
```

Comment:
Seeing strange behavior here where when this isn't provided, the value is never set to false if it was previously set to true.